.changeset/effect-beta-101.md

```
@@ -0,0 +1,16 @@
+'foldkit': patch
+'@foldkit/ui': patch
+'@foldkit/devtools': patch
+'@foldkit/devtools-mcp': patch
+'@foldkit/vite-plugin': patch
+---
+
+Bump Effect to `4.0.0-beta.101` (from `4.0.0-beta.97`). Foldkit's peer dependencies now require `effect@4.0.0-beta.101` and `@effect/platform-browser@4.0.0-beta.101`.
+
+Consumers should align their Effect packages to `4.0.0-beta.101` exactly during the v4 beta window:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)